test/sessionEventAndCoC.js

```diff
@@ -0,0 +1,273 @@
+
+'use strict';
+
+const assert = require('chai').assert;
+let generator = require('../src/backend/generator.js');
+let app = require('../src/app');
+let webdriver = require('selenium-webdriver');
+let sessionPage = require('../src/selenium/sessionPage.js');
+let eventPage = require('../src/selenium/eventPage.js');
+let cocPage = require('../src/selenium/cocPage.js');
+
+describe('app', () => {
+  describe('run', () => {
+    it("should run app", () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)